Iterable/iterable-web-sdk
# [MOB-6229] Reposition close x button

## JIRA Ticket(s) if any

* [MOB-6229](https://iterable.atlassian.net/browse/MOB-6229)

## Description

Close button was being positioned relative to document rather than iframe boundaries.
For non-Safari browsers, return close x button to the iframe document.
For Safari browsers, keep close button on the main document and dynamically reposition relative to iframe boundaries.

## Test Steps

In both Safari and non-Safari browsers:
1. Send in-app to self
2. Call getMessages in application
3. Expected: close x button appears over message iframe according to closeButton params
4. Expected: resizing window keeps close x button over message iframe

[MOB-6229]: https://iterable.atlassian.net/browse/MOB-6229?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
